src/components/Swap/Summary/index.tsx

```diff
@@ -45,25 +44,22 @@ function useReviewState(onSwap: () => Promise<void>, allowance: Allowance, doesT
       }
       // if the user finishes permit2 allowance flow, onStartSwapFlow() will be called again by useEffect below to trigger swap
     } else if (allowance.state === AllowanceState.ALLOWED) {
-      if (doesTradeDiffer) {
-        setCurrentState(ReviewState.TRADE_CHANGED)
+      // Prevents immeadiate swap if trade has updated mid permit2 flow
```

Comment:
typo

Reply:
i swear I thought immediate was spelled that way wow. thank you